### PR TITLE
Munge Wisconsin

### PR DESCRIPTION
**Pull request by EKMeyerson**
Add juris files for Wisconsin, munger files (wi_pri) were already there :)

Notes:
- Used single munger (wi_pri) for all files, both primary and general.
- Munged at county level. Ward level is available, but more difficult to munge, and I think not a priority.
- In statewide 2018 elections, write-ins for gov./lt. gov. where one of gov./lt.gov is "No Candidate" are not munged correctly, maybe due to line break. These candidates should not be important, so ignoring them.
- Have not added contests from Spring 2020 election concurrent with presidential primary, e.g., WI Supreme Court and Court of Appeals. Can add these if interesting.

### CODE REVIEW

**Comment by sfsinger19103** [comment]
What tests (if any) did you run @EKMeyerson ?

**Review by sfsinger19103 — CHANGES_REQUESTED** [review]
After rebasing on `main`, please:

- Write three test files, one for each election. You can use as a model `test/test_Florida_2016-General.py` as a model
- Using (new) documentation in UserGuide.md, use `load_and_test_all.py` to run the tests for each election, making any necessary corrections.

I tried testing 2016 General, and found that many candidates were not recognized. Did you have a different experience?
```Candidates not found in dictionary.txt:
Tom Hoefling
Steve Schulin (write-in)
Rocky Roque De La Fuente
Michael Steinberg
Monica Moorehead
Lamont Lilly
Michael A. Maturen
Juan Munoz (write-in)
Marshall Schoenke
James Creighton Mitchell Jr. (write-in)
Laurence Kotlikoff
Edward E. Leamer (write-in)
Jill Stein
Ajamu Baraka
Hillary Clinton
Tim Kaine
Gary Johnson
Bill Weld
Evan McMullin
Nathan Johnson (write-in)
Emidio Soltysik
Angela Nicole Walker (write-in)
Donald J. Trump
Michael R. Pence
Darrell L. Castle
Scott N. Bradley
Chris Keniston
Deacon Taylor (write-in)
Cherunda Fox
Roger Kushner (write-in)```

**Comment by sfsinger19103** [comment]
Our standard for joint contests (Pres & VP, Gov & Lt.Gov) is to use the top contest and candidate internally. E.g., the Contest.Name is 'US President (WA)' and the Candidate.BallotName is 'Donald J. Trump'.

File: src/mungers/wi_pri/cdf_elements.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@ name	raw_identifier_formula	source
 ReportingUnit	<first_column>	row
 Party	<header_0>	row
 CandidateContest	<constant_line_0>	row
```
Comment (EKMeyerson) [inline src/mungers/wi_pri/cdf_elements.txt:4]: Here's the regex.

Comment (sfsinger19103) [reply src/mungers/wi_pri/cdf_elements.txt:4]: @EKMeyerson is it ready for me to test now? Please tag me in your response so GitHub will notify me.

**Comment by EKMeyerson** [comment]
@sfsinger19103, the test files have been added and pass. The only hiccup is vote total that was off by 3 for the 2018 governor's race. My hypothesis is that it's because there are two write-in candidates for whom the governor's name is "No Candidate", the latter of which has 3 votes and gets dropped. I've included a comment in the test for now.

**Comment by sfsinger19103** [comment]
Please add "No Candidate" to Candidate.txt and dictionary.txt so that everything adds up. [For all we know, maybe someone changed their first name to "No" and their last name to "Candidate".]

**Review by sfsinger19103 — APPROVED** [review]

**Comment by EKMeyerson** [comment]
> Please add "No Candidate" to Candidate.txt and dictionary.txt so that everything adds up. [For all we know, maybe someone changed their first name to "No" and their last name to "Candidate".] @EKMeyerson

@sfsinger19103 "No Candidate" is in both of those. The issue is that there's a single contest where "No Candidate" appears twice in the list of candidates (munge-10-Wisconsin/Results and Parameter Files/Wisconsin_20181106/County by County Report-2018 Gen Election-State Constitutional Offices_1.xlsx; Sheet 2), and only the first occurrence is getting added to the database.

**Pull request merged**